src/style.css

```diff
@@ -0,0 +1,75 @@
+* {
+  box-sizing: border-box;
+}
+@font-face {
+  font-family: avenir;
+  src: url(../src/fonts/Avenir.ttc), url(../src/fonts/Avenir-Heavy.ttf), url(../src/fonts/Avenir-Book.ttf);
+}
+
+.search-bar {
+  width: 100%;
+  font-family: avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(../src/images/Search.svg);
+  background-repeat: no-repeat;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.space {
+  height: 12px;
+}
+
+.space1 {
+  height: 20px;
+}
+
+.search1 {
+  height: 70px;
+  background-position: 25px 26px;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.search2 {
+  height: 42px ;
+  background-position-x: 13px;
+  background-position-y: 50%;
+  background-size: 11px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.search-bar::placeholder {
+  font-family: avenir, Arial, Helvetica, sans-serif;
+}
+
+.search-bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border: 1px solid #e1e7;
+  border-radius: 4px;
+
+}
+
+.search-bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-image: url(../src/images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 26px;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.search1:focus::placeholder {
+  font-weight: 900;
```

Comment:
i thing you should add 
`color: #3d4e61;` here
and remove `font-weight: 900;` from here